<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JiRaTool</class>
 <widget class="QMainWindow" name="JiRaTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>287</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1121</width>
    <height>287</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QPushButton" name="prev_btn">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Prev</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="next_btn">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="auth_btn">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_btn">
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 255, 0);
background-color: rgb(170, 170, 255);</string>
        </property>
        <property name="text">
         <string>Refresh Board</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="jira_task_list_table">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <property name="columnCount">
       <number>11</number>
      </property>
      <column>
       <property name="text">
        <string>Ticket</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Mon</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tue</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>170</green>
         <blue>127</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Wed</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Thu</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fri</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sat</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sun</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="background">
        <color>
         <red>142</red>
         <green>255</green>
         <blue>231</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>StoryPoints</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time Spent</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
